sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpSender.cs

```diff
@@ -224,12 +222,12 @@ internal class AmqpSender : TransportSender
         ///    Sends a set of messages to the associated Queue/Topic using a batched approach.
         /// </summary>
         ///
-        /// <param name="messageFactory"></param>
+        /// <param name="messages"></param>
         /// <param name="timeout"></param>
         /// <param name="cancellationToken">An optional <see cref="CancellationToken"/> instance to signal the request to cancel the operation.</param>
         ///
         internal virtual async Task SendBatchInternalAsync(
```

Comment:
This is internal virtual, so I wasn't sure if I'm allowed to change it or not. Please advise

Reply:
Yes feel free to change.